data/src/main/java/me/androidbox/data/remote/model/response/FullAgendaModel.kt

```diff
@@ -0,0 +1,7 @@
+package me.androidbox.data.remote.model.response
+
+data class FullAgendaModel(
+    val listOfEventId: List<String>,
+    val listOfTaskId: List<String>,
+    val listOfReminderId: List<String>
+)
```

Comment:
The names of the fields need to match the names of the JSON, otherwise you'll get a crash